server/routes/homeStatusSection.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+
+const router = express.Router();
+const log = require('../helper/logger');
+
+const WishCardRepository = require('../db/repository/WishCardRepository');
+const AgencyRepository = require('../db/repository/AgencyRepository');
+
+// Renders data to homepage on sats section.
+
+router.get('/', async (req, res) => {
+  try {
+    const wCards = await WishCardRepository.getNotDonatedCards();
```

Comment:
I agree with Stacy, since this is just for one route, creating a separate file is bit overkill. Best to move the code into https://github.com/donategifts/donategifts/blob/cce033e834db222855ddd6c98c2d0762aa1dfe51/server/app.js#L187

Also, I believe this implementation is being overridden by the default home render from app.js (which does use user and wishcards).